easybuild/easyblocks/w/wps.py

```diff
@@ -251,7 +259,12 @@ def run_wps_cmd(cmdname, mpi_cmd=True):
             if not self.cfg['testdata']:
                 raise EasyBuildError("List of URLs for testdata not provided.")
 
-            wpsdir = os.path.join(self.builddir, "WPS")
+
+            mainver = self.version.split('.')[0]
+            if mainver < 4:
```

Comment:
This does not do what you think it does :)

You’re comparing a string with an integer here. That doesn’t cause any type errors in Python 2.x, but you really don’t want to do that since this will always be False regardless of the value of mainver...

Please also use LooseVersion here, and drop the ‘split’ above, compare using self.version

Reply:
addressed by https://github.com/andreas-h/easybuild-easyblocks/pull/4